apps/web/src/components/StatusItem/StatusItem.tsx

```diff
@@ -106,10 +126,8 @@ const StatusItem = ({
   return (
     <Wrapper
       data-h2-display="base(flex)"
-      data-h2-border-top="base:selectors[:not(:first-child)](1px solid gray.lighter)"
-      data-h2-margin-top="base:selectors[:not(:first-child)](x.5)"
-      data-h2-padding-top="base:selectors[:not(:first-child)](x.5)"
       data-h2-justify-content="base(space-between)"
+      {...layoutStyleMap[layout]}
```

Comment:
Can we add a tiny `gap` on this to space the main text out a bit from the item count?

Reply:
Good idea, yeah.